packages/ember-simple-auth-oauth2/lib/simple-auth-oauth2/configuration.js

```diff
@@ -24,6 +25,17 @@ var defaults = {
 */
 export default {
   /**
+    The client_id sent to the server
```

Comment:
There should be a warning here saying that the client id cannot actually be trusted and should only be used for statistics purposes on the server side. I think this might mislead a lot of people otherwise.